RiccardoMolinari95/io-app
# Feature flusso cancellazione profilo

Onboarding - Flusso Cancellazione Profilo

## Short description
Added flow to delete profile, via two possibilities:
- added a button "Elimina il tuo profilo" in “Portafoglio”
- by moving the switch from “off” to “on” on the new profile page.

## How to test
Delete the profile via the newly added button and continue with the deletion via the various confirmation screens. Once deleted , go to the new profile page and see if the loader is loading and the switch is "on".
Return to the “Portfolio” section and click the “Retrieve Profile” button to cancel the deletion request by accepting the alert
